### DISCUSSION TITLE
Ignore multiple files

### DISCUSSION THREAD

**Discussion by moritzbaettig** [pgrimaud/action-shopify#11] [Q&A] [answered]
Hi, does anybody know how it is possible to ignore multiple files?
I know that the following line in the workflow.yml file would cause the file "a.liquid" to be ignored:

args: --ignored-file=a.liquid

But what if I want to ignore "a.liquid" AND "b.liquid"? How does the argument look like?

**Comment by pgrimaud** [comment, marked as answer]
Hi, 

Yes you can add multiple times the `--ignored-file` option. 


Example : `args: --ignored-file=a.liquid --ignored-file=b.liquid`

**Reply by moritzbaettig** [reply]
Thank you!

**Reply by pgrimaud** [reply]
If my answer resolved your question, can you check it as « Mark as answer »? It could help other people who may have the same question. 

Thanks ✌️
